content/v2/oauth.markdown

```diff
@@ -16,15 +16,63 @@ Developers need to register their application before getting started. A register
 
 ## Web Application Flow
 
-This is a description of the OAuth 2 flow for authorizing users from 3rd party web sites. The OAuth 2 specification is described in the [RFC6749](http://tools.ietf.org/html/rfc6749#section-4.1).
+This is a description of the OAuth 2 flow for authorizing users from 3rd party web sites. The OAuth 2 specification is described in the [RFC 6749](http://tools.ietf.org/html/rfc6749#section-4).
+
+<info>
+The API currently supports only the OAuth 2 [Authorization Code Grant](http://tools.ietf.org/html/rfc6749#section-4.1) flow. Therefore, [the only supported value for the `response_type` is `code`](http://tools.ietf.org/html/rfc6749#section-3.1.1).
+</info>
 
 ### Step 1 - Authorization
 
+This starts the OAuth 2.0 authorization flow. This isn't an API call — it's the web page that lets the user sign in to DNSimple and authorize your app. After the user decides whether or not to authorize your app, they will be redirected to the URI specified by `redirect_uri`.
+
+Your web or mobile app should redirect users to the following URL:
+
+~~~
+GET https://dnsimple.com/oauth/authorize
+~~~
+
+The following values should be passed as GET parameters:
+
+`response_type` | **Required**. The grant type requested. We currently only support `code`.
+`client_id`     | **Required**. The client ID you received from DNSimple when you registered the application.
+`redirect_uri`  | Where to redirect the user after authorization has completed. This must be the exact URI registered or a subdirectory.
+`scope`         | We currently don't use this field.
+`state`         | An unguessable random string. It is used to protect against cross-site request forgery attacks and it will be passed back to your redirect URI.
+
+Because `/oauth/authorize` is a website, there is no direct return value. However, after the user authorizes your app, they will be sent to your redirect URI.
```

Comment:
@jacegu your sentence is a little bit misleading. In theory, you can leave the `redirect_uri` parameter out and we'll use the URI you registered when you registered the app.